siriuspy/siriuspy/devices/beamline.py

```diff
@@ -0,0 +1,77 @@
+"""Beamline devices."""
+
+from .device import Device as _Device
+
+
+class BeamLineMirror(_Device):
```

Comment:
I think this is a device very specific for CAX, the names of the PVs seems to be different for each beamline

Reply:
I thought they all followed a standard for mirror controls...

Reply:
See what you think now.